### ISSUE TITLE
`mwoffliner --version` delivers the wrong version number

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#978]
I have installed MWoffliner 1.9.9 with:
```
sudo npm install -g mwoffliner@1.9.9
```

but it tells me I have the 1.9.6 version:
```
$mwoffliner --version
1.9.6
```

**Comment by kelson42** [comment]
@ISNIT0 Not sure this is a problem with my system or a bug, I'm just unable to get it works fine.

**Comment by ISNIT0** [comment]
I'm seeing the same. Perhaps there's a bug in the auto publishing...

**Comment by ISNIT0** [comment]
This is fixed in the latest release (1.9.10)
